src/component/movie-components/Comments/AddComment.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+import { connect } from "react-redux";
+import { createComment } from "../../../store/actions/CommentActions";
+
+class AddComment extends React.Component {
+  constructor() {
```

Comment:
Remove constructor if you are only setting state in it, since state can be set in component itself.
```javascript
class AddComment extends React.Component {
  state = {
    ...
  }
...
}
```